adafruit_emc2101/emc2101_lut.py

```diff
@@ -97,10 +124,21 @@ def __str__(self):
         lut_keys = tuple(sorted(self.lut_values.keys()))
         for temp in lut_keys:
             fan_drive = self.lut_values[temp]
+            # pylint: disable=consider-using-f-string
             value_strs.append("%d deg C => %.1f%% duty cycle" % (temp, fan_drive))
```

Comment:
Would be ideal if we can change this over to f-string instead of disabling the check for it.

Reply:
I think f-string is disabled on some of the smaller boards, should we use `.format()`?  I thought the CI prevented this warning from being raised by pylint, which is why I asked.  I think something similar happened to me recently.

Reply:
Ah, yeah `.format()` would be best then I think.